tox.ini

```
@@ -19,8 +19,19 @@ commands =
 select = F401,F402,F821,F823,F841,F811,E265,E266,F541,W605,E722,F822,F523,W291,F405
 exclude = awx/ui/node_modules,awx/ui/node_modules,env,awx_collection_build
 
+[testenv:pip-compile-docs]
+description = Compile docs build requirements
+deps =
+  # pyproject.toml was introduced in v7
```

This env could also have its own set of constraints. This could be introduced in follow-ups, of course.

Let's fix the accuracy here:
```suggestion
  # pip-tools config file support was introduced in v7
```